RELEASE_NOTES.md

```diff
@@ -0,0 +1,70 @@
+# Performance Optimizations
+## Intel Architecture Processors
+* Improved matmul performance on Intel Xeon processors with Intel AMX instruction set support.
+* Improved `fp16`/`bf16` depthwise convolution performance with `fp32` bias or `sum` post-op.
+* Improved `bf16` pooling backpropagation performance.
+* Improved binary post-op performance with `per_w` broadcast.
+
+## Intel Graphics Products
+* Improved performance for Intel GPUs based on Xe3 architecture.
+* Improved convolution performance on Intel Arc Graphics for Intel Core Ultra processors (Series 2) (formerly Lunar Lake) and Intel Arc B-series discrete graphics (formerly Battlemage).
+* Improved `int8` matmul performance with zero points for source and weight tensors.
+* Improved performance of the following subgraphs with Graph API:
+	* Scaled Dot Product Attention (SDPA) with `int4` and `int8` KV cache.
+	* SDPA with bottom-right implicit causal mask.
+	* SDPA with head size between 257 and 512.
+	* Grouped Query Attention (GQA) with 5D input tensors.
+
+## AArch64-based Processors
+* Enabled BF16 forward-mode inner product via ACL. 170x and 160x speedups for BERT and AlexNet in torch compile-mode.
```

Comment:
not sure if i understand it correctly, but if i do, i would try to make it a little bit clearer. Maybe smth like:

* Enabled BF16 forward-mode inner product via ACL with the speedups:
   * BERT: up to 170x
   * AlexNet: up to 160x (torch compile mode)

Reply:
I'd suggest dropping the second sentence alltogether as it refers to PyTorch and not sufficiently qualified.